atom/browser/api/atom_api_web_contents.cc

```diff
@@ -971,6 +989,50 @@ void WebContents::AuthorizePlugin(mate::Arguments* args) {
       web_contents(), true, resource_id);
 }
 
+void WebContents::AddWord(mate::Arguments* args) {
```

Comment:
the spellcheck service is associated with the browser context, not the webcontents so this should probably be an api attached to session like autofill